atomic_reactor/plugins/prepub_squash.py

```diff
@@ -57,14 +55,13 @@ class PrePublishSquashPlugin(PrePublishPlugin):
     is_allowed_to_fail = False
 
     def __init__(self, tasker, workflow, tag=None, from_base=True, from_layer=None,
-                 remove_former_image=True, dont_load=False):
+                 dont_load=False):
```

Comment:
I don't think we can just remove it from the parameter list -- older osbs-clients will generate build JSON that will no longer work.